apps/lfx-one/src/app/modules/committees/committee-dashboard/committee-dashboard.component.html

```
@@ -26,10 +26,156 @@ <h1>{{ committeeLabelPlural }}</h1>
+        <div class="flex items-center gap-2 mb-4">
+          <h2 class="text-lg font-semibold text-gray-900">My {{ committeeLabelPlural }}</h2>
+        </div>
+        <div class="flex items-center gap-2 text-gray-400 text-sm py-4">
+          <i class="fa-light fa-spinner-third fa-spin"></i>
+          <span>Loading your groups...</span>
+        </div>
+      </div>
+    }
+
+    <!-- All Groups Section -->
+    @if (myCommittees().length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)